src/elements/Bookmark/Bookmark.slate.stories.tsx

```diff
@@ -0,0 +1,306 @@
+import React from 'react';
+import type { Meta, Story } from '@storybook/react';
+import createAsyncCallback from '@loki/create-async-callback';
+import { Renderer } from '../../Renderer';
+
+export default {
+    title: 'Elements/Bookmark',
+    decorators: [
+        (Story) => {
+            const cb = createAsyncCallback();
+
+            setTimeout(() => {
+                cb();
+            }, 1000);
+
+            return <Story />;
```

Comment:
Looks weird. What is it supposed to do exactly? 🤔 
Does it simulate a 1 second delay before Story is returned?

Reply:
It delays 1s before screenshot capture, not the story render